pyomo/solvers/plugins/solvers/SCIPAMPL.py

```diff
@@ -199,15 +199,76 @@ def create_command_line(self, executable, problem_files):
         return Bunch(cmd=cmd, log_file=self._log_file, env=env, cwd=options_dir)
 
     def _postsolve(self):
-        results = super(SCIPAMPL, self)._postsolve()
-        if results.solver.message == "unknown":
+        # results = super(SCIPAMPL, self)._postsolve() # previous way
+        # copied super(SCIPAMPL, self)._postsolve() to access the log file
+        if self._log_file is not None:
+            OUTPUT=open(self._log_file,"w")
+            OUTPUT.write("Solver command line: "+str(self._command.cmd)+'\n')
+            OUTPUT.write("\n")
+            OUTPUT.write(self._log+'\n')
+            OUTPUT.close()
+
+        # JPW: The cleanup of the problem file probably shouldn't be here, but
+        #   rather in the base OptSolver class. That would require movement of
+        #   the keepfiles attribute and associated cleanup logic to the base
+        #   class, which I didn't feel like doing at this present time. the
+        #   base class remove_files method should clean up the problem file.
+
+        if (self._log_file is not None) and \
+           (not os.path.exists(self._log_file)):
+            msg = "File '%s' not generated while executing %s"
+            raise IOError(msg % (self._log_file, self.path))
+        results = None
+
+        if self._results_format is not None:
+            results = self.process_output(self._rc)
+            if (self._log_file is not None) and \
+               (os.path.exists(self._log_file)):
+                   
+               log_dict = self.read_scip_log(self._log_file)
+            
+               results.solver.time = log_dict['solving_time']
+               results.solver.gap = log_dict['gap']
+               results.solver.primal_bound = log_dict['primal_bound']
+               results.solver.dual_bound = log_dict['dual_bound']
+               
+               # TODO: get scip to produce a statistics file and read it
+               # Why? It has all the information one can possibly need.
+                   
+            #
+            # If keepfiles is true, then we pop the
+            # TempfileManager context while telling it to
+            # _not_ remove the files.
+            #
+            if not self._keepfiles:
+                # in some cases, the solution filename is
+                # not generated via the temp-file mechanism,
+                # instead being automatically derived from
+                # the input lp/nl filename. so, we may have
+                # to clean it up manually.
+                if (not self._soln_file is None) and \
+                   os.path.exists(self._soln_file):
+                    os.remove(self._soln_file)
+
+        TempfileManager.pop(remove=not self._keepfiles)
+        
+        #**********************************************************************
+        
+        # # UNKNOWN
+        # unknown='unknown'                               # An unitialized value
```

Comment:
I appreciate these comments being inserted, but can they please be cleaned up a bit? I.e., one comment per line? (Also, note, typo in this particular comment)
```
## UNKNOWN
# unknown=`unknown`
# An uninitialized value 
```